packages/oui-numeric/src/numeric.controller.js

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash'
+
+// By design, value is restricted to [0, 99999] interval
+const MIN_VALUE = 0
+const MAX_VALUE = 99999
+
+export default class {
+  constructor ($attrs, $element, $log) {
+    'ngInject'
+
+    this.$attrs = $attrs
+    this.$element = $element
+    this.$log = $log
+  }
+
+  $postLink () {
+    this.$element.removeAttr('id')
+    this.$element.removeAttr('name')
+  }
+
+  $onInit () {
+    if (!this.id) {
+      this.$log.warn(`Missing required attribute id`)
+    }
+
+    if (!this.text) {
+      this.$log.warn(`Missing required attribute text`)
+    }
+
+    if (!angular.isNumber(this.min)) {
+      if (angular.isDefined(this.min)) {
+        this.$log.warn(`Invalid attribute min, expected number got '${this.min}'`)
+      }
+      this.min = MIN_VALUE
+    }
+
+    if (!angular.isNumber(this.max)) {
+      if (angular.isDefined(this.max)) {
+        this.$log.warn(`Invalid attribute max, expected number got '${this.max}'`)
+      }
+      this.max = MAX_VALUE
+    }
+
+    if (!angular.isNumber(this.model)) {
+      if (angular.isDefined(this.model)) {
+        this.$log.warn(`Invalid attribute model, expected number got '${this.model}'`)
+      }
+      // if model is undefined, initialize it with min value
+      this.setModelValue(this.min)
+    }
+
+    if (this.min < MIN_VALUE) {
+      this.$log.warn(`Invalid attribute min, value should be greater than '${MIN_VALUE}'`)
+    }
+
+    if (this.max < MAX_VALUE) {
```

Comment:
wrong direction :) should be `this.max > MAX_VALUE`

You don't test your warn messages ;)